### PR TITLE
Improve node-scenario docs

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds sample configuration files for each of the supported platforms.

### CODE REVIEW

File: docs/node_scenarios.md [hunk 1]
```diff
@@ -60,9 +71,12 @@ The supported node level chaos scenarios on an OPENSTACK cloud are `node_stop_st
 To execute the scenario, ensure the value for `ssh_private_key` in the node scenarios config file is set with the correct private key file path for ssh connection to the helper node. Ensure passwordless ssh is configured on the host running Kraken and the helper node to avoid connection errors.
 
 
+
 #### Azure
 
-How to set up Azure cli to run node scenarios is defined [here](cloud_setup.md#azure).
+How to set up Azure cli to run node scenarios is defined [here](cloud_setup.md#azure). Sample scenario config can be found [here](https://github.com/krkn-chaos/krkn/blob/main/scenarios/openshift/azure_node_scenarios.yml).
```
Comment (paigerube14) [inline docs/node_scenarios.md:77]: I think I've actually removed all need for the azure cli now, so think we can take out points around that

Same with gcp cli

File: scenarios/openshift/node_scenarios_example.yml [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+    cloud_type: aws                                                 # Cloud type on which Kubernetes/OpenShift runs.
   - actions:
     - node_reboot_scenario
     node_name:
     label_selector: node-role.kubernetes.io/infra
     instance_count: 1
     timeout: 120
-    cloud_type: aws
+    cloud_type: azure
+  - actions:
```
Comment (paigerube14) [inline scenarios/openshift/node_scenarios_example.yml:19]: this seems a little confsing to me to ahve both azure and baremetal in one file, should we separate into two separate files and then link to each from the node doc

File: docs/node_scenarios.md (hunk 1 above)
Comment (chaitanyaenr) [reply docs/node_scenarios.md]: Updated, thanks!

File: scenarios/openshift/node_scenarios_example.yml (hunk 2 above)
Comment (chaitanyaenr) [reply scenarios/openshift/node_scenarios_example.yml]: Done. Thanks.

File: docs/node_scenarios.md [hunk 3]
```diff
@@ -19,14 +19,17 @@ The following node chaos scenarios are supported:
 , node_reboot_scenario and stop_start_kubelet_scenario are supported only on AWS, Azure, OpenStack, BareMetal, GCP
 , VMware and Alibaba as of now.
 
-**NOTE**: Node scenarios are supported only when running the standalone version of Kraken until https://github.com/redhat-chaos/krkn/issues/106 gets fixed.
-
 
 #### AWS
 
-How to set up AWS cli to run node scenarios is defined [here](cloud_setup.md#aws).
+How to set up AWS cli to run node scenarios is defined [here](cloud_setup.md#aws). Sample scenario config can be found [here](https://github.com/krkn-chaos/krkn/blob/main/scenarios/openshift/aws_node_scenarios.yml).
```
Comment (paigerube14) [inline docs/node_scenarios.md:25]: same thing here, we dont need the cli I believe

File: docs/node_scenarios.md [hunk 4]
```diff
@@ -46,8 +51,11 @@ The Docker provider can be used to run node scenarios against kind clusters.
 
 kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
 
+
+
 #### GCP
-How to set up GCP cli to run node scenarios is defined [here](cloud_setup.md#gcp).
+Sample scenario config can be found [here](https://github.com/krkn-chaos/krkn/blob/main/scenarios/openshift/gcp_node_scenarios.yml).
```
Comment (paigerube14) [inline docs/node_scenarios.md:57]: let's still link to each of the cloud setup details that was here but exclude the "cli" details

File: docs/node_scenarios.md [hunk 5]
```diff
@@ -73,43 +83,22 @@ How to set up Alibaba cli to run node scenarios is defined [here](cloud_setup.md
...
 *vmware-node-terminate, vmware-node-reboot, vmware-node-stop, vmware-node-start*
 
+
+
 #### IBMCloud
 How to set up IBMCloud to run node scenarios is defined [here](cloud_setup.md#ibmcloud)
 
 This cloud type uses a different configuration style, see actions below and [example config file](../scenarios/openshift/ibmcloud_node_scenarios.yml)
 
 *ibmcloud-node-terminate, ibmcloud-node-reboot, ibmcloud-node-stop, ibmcloud-node-start
```
Comment (paigerube14) [inline docs/node_scenarios.md:101]: maybe lets make this a list with each scenario type per line, would be easier to read

**Comment by chaitanyaenr** [comment]
@paigerube14 updated the commit with the suggested changes. PTAL. Thanks.

File: docs/node_scenarios.md [hunk 6]
```diff
@@ -73,43 +83,28 @@ How to set up Alibaba cli to run node scenarios is defined [here](cloud_setup.md
...
 #### VMware
 How to set up VMware vSphere to run node scenarios is defined [here](cloud_setup.md#vmware)
 
 This cloud type uses a different configuration style, see actions below and [example config file](../scenarios/openshift/vmware_node_scenarios.yml)
 
-*vmware-node-terminate, vmware-node-reboot, vmware-node-stop, vmware-node-start*
+- vmware-node-terminate
+- vmware-node-reboot
+- vmware-node-stop
+- vmware-node-start*
```
Comment (paigerube14) [inline docs/node_scenarios.md:95]: minor nit, take off * at end of line

**Review by paigerube14 — APPROVED** [review]
lgtm

**Pull request merged**